eppo_client/client.py

```diff
@@ -261,13 +261,15 @@ def get_bandit_action(
             actions (ActionContexts | ActionAttributes): The dictionary that maps action keys
                 to their context of actions with their contexts.
                 If supplying an ActionAttributes, it gets converted to an ActionContexts instance.
-            default (str): The default variation to use if the subject is not part of the bandit.
+            default (str): The default variation to use if an error is encountered retrieving the
+                assigned variation.
 
         Returns:
             BanditResult: The result containing either the bandit action if the subject is part of the bandit,
                           or the assignment if they are not. The BanditResult includes:
                           - variation (str): The assignment key indicating the subject's variation.
-                          - action (str): The key of the selected action if the subject is part of the bandit.
+                          - action (str | null): The key of the selected action if the subject was assigned one
```

Comment:
returned actions can be None

Reply:
Nit: should say `None` instead of `null`, or better: `Optional[str]` to keep it Pythonic